descriptors/ModuleDescriptor-template.json

```diff
@@ -240,7 +240,7 @@
     },
     {
       "id": "finance.ledgers",
-      "version": "1.2",
+      "version": "1.3",
       "handlers": [
         {
           "methods": ["GET"],
```

Comment:
Please add `finance-storage.ledgersFY.collection.get` to modulePermissions for `GET /finance/ledgers` handler